x/oracle/types/msgs.go

```diff
@@ -0,0 +1,62 @@
+package types
+
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+)
+
+var _, _ sdk.Msg = &MsgUpdateOracleRequest{}, &MsgSendQueryOracleRequest{}
```

Comment:
Please split this onto multiple lines and put them in the same order that they're in in this file.
```suggestion
var (
	_ sdk.Msg = &MsgSendQueryOracleRequest{}
	_ sdk.Msg = &MsgUpdateOracleRequest{}
)
```

Alternatively, put them in a slice like we have in a couple modules (e.g. [Metadata](https://github.com/provenance-io/provenance/blob/main/x/metadata/types/msg.go#L49-L80)). Then that slice can be used in places where we always need to remember to add new message type definitions, and we've reduced possible pain points.